app/mapper/ReservationMapper.py

```diff
@@ -52,9 +52,6 @@ def findAll():
                 ReservationIdMap.addTo(reservation)
     return allReservations
 
-def findByDate(date):
```

Comment:
It was not being used anywhere, and it was only returning raw data, not reservations. Where you planning on using it?